swift/ql/src/queries/Summary/SummaryStats.ql

```diff
@@ -0,0 +1,27 @@
+/**
+ * @name Summary statistics
+ * @description A table of summary statistics about a database. Includes
+ *              values that measure its size, and the numbers of certain
+ *              features interesting to analysis that have been found.
```

Comment:
I have a very hard time parsing "and the numbers of certain features interesting to analysis that have been found". What does "that have been found" refer to? "features" or "analysis"? Maybe just drop that part? I don't think that's quite sufficient for me to understand what you're trying to say, but it's a step.

Reply:
You're right.  I've removed it.  Keep it simple.